iina/MainWindowController.swift

```diff
@@ -1056,6 +1056,11 @@ class MainWindowController: NSWindowController, NSWindowDelegate {
       player.togglePause(nil)
     case .hideOSC:
       hideUI()
+    case .togglePIP:
+      if #available(OSX 10.12, *) {
+        togglePIP()
```

Comment:
You should directly use `menuTogglePIP` and pass a `.dummy`.

Reply:
I'm sorry, I couldn't follow your reasoning here. Could you expand on it?

Is it the refactoring of common functionality from `menuTogglePIP` into `togglePIP` that you object to? It's done to preserve clear semantics of symbol names, since toggling pip isn't done exclusively from menus. I find it makes intention clearer than passing superfluous dummy objects.

Reply:
Sorry I didn't see your refactoring when I reviewed your pr. Your refactoring is great, but we are currently using menu actions with dummy menu item everywhere. So I believe we will take this into account when we do the next major refactoring.